### PR TITLE
Rename exogains vars

### PR DESCRIPTION
**Pull request by robinhasse**
Buildings emissions from exogains used an inconsistent sector name (`ResCom`) which is now aligned with all other buildings variables. I also changed quotes in .buildlibrary as the old quotes caused `lucode2::buildlibrary()` to fail. @fbenke-pik, will theses quotes also work on ubuntu? I didn't check that so far.

### CODE REVIEW

**Comment by merfort** [comment]
Thanks Robin! Looks all good to me.
Just a note: These `ResCom` variables are also found in other repositories, in particular in `project_interfaces` (I used `reposearch` on the cluster):
```
repos/pik-piam/project_interfaces/master/navigate/mapping_template_NAVIGATE.csv
repos/pik-piam/project_interfaces/master/ar6/mapping_template_AR6.csv
repos/pik-piam/project_interfaces/master/ar6/shape/shape_template.csv
```
I guess we won't touch AR6 mappings anymore, but for NAVIGATE that might be relevant.

**Comment by merfort** [comment]
@nicobauer If we rename these variables, will this interfere with the NAVIGATE mapping? Or more specific: Will there be new NAVIGATE runs or at least an update of the reporting? That would then be the time to also update `repos/pik-piam/project_interfaces/master/ar6/shape/shape_template.csv`

**Comment by fbenke-pik** [comment]
@merfort Latest mapping should now be in piamInterfaces, project_interfaces will be deactivated soon. In future, we will have a test to check consistency with NAVIGATE, but for now you still have to check manually for all project templates not covered by tests yet in the piamInterfaces repo

File: .buildlibrary [hunk 1]
```diff
@@ -8,6 +8,6 @@ AcceptedNotes:
 - Imports includes .* non-default packages.
 - unable to verify current time
 - checking installed package size
-- Namespace in Imports field not imported from: ‘piamInterfaces’
+- Namespace in Imports field not imported from: piamInterfaces
```
[the remark this answers is not in the document]
Comment (robinhasse) [reply .buildlibrary]: Thanks Falk, I corrected this.

**Comment by fbenke-pik** [comment]
Quick search of files in piamInterfaces repo reveals that for now, none of the templates in the repo uses "ResCom", so we are good for now. It is still used in Ariadne, though. I will take care of it once I migrate the template into piamInterfaces.

**Review by fbenke-pik — APPROVED** [review]

**Review by merfort — APPROVED** [review]
Thanks Falk! Then we are good to go from my side :)

**Pull request merged**